backend/src/main/java/com/jobwait/stat/Stats.java

```diff
@@ -0,0 +1,35 @@
+package com.jobwait.stat;
+
+import java.util.List;
+
+import org.springframework.http.HttpStatus;
+
+import com.jobwait.domain.Stat;
+import com.jobwait.fault.FaultException;
+import com.jobwait.stat.impl.JobTitleStat;
+
+public class Stats {
+    // touch this stuff
+    public static List<Stat> known = List.of(
+            new JobTitleStat("job-title"));
```

Comment:
"job-title" should be defined inside JobTitle, it feels redundant to pass it as an argument to the class of the same name. I did this on purpose because if you want to know at a glance what id is used for a certain stat, you can go to Stats and see them all here. If I put it in the class, you have to go into each class to see their ids. I think the convenience is worth the redundancy, but let me know your thoughts.

Reply:
I think that's fair. Moreover, I think it would be useful if potentially (I don't know why) we had the same stat but with a different id. Either way I think its ok for now at least.